src/fhirdefs/impliedExtensions.ts

```diff
@@ -0,0 +1,457 @@
+// This file contains logic to support "implied" extensions, described in the FHIR spec here:
+// http://hl7.org/fhir/versions.html#extensions
+
+import { union } from 'lodash';
+import { logger, Type } from '../utils';
+import { ElementDefinition, ElementDefinitionType, StructureDefinition } from '../fhirtypes';
+import { FHIRDefinitions } from '../fhirdefs';
+
+export const IMPLIED_EXTENSION_REGEX = /^http:\/\/hl7\.org\/fhir\/([1345]\.0)\/StructureDefinition\/extension-(([^./]+)\.[^\/]+)$/;
```

Comment:
> I'm trying to figure out why in (([^./]+)\.[^\/]+), the first / does not need to be escaped. But it seems like it doesn't...but it still scares me since ole reliable https://regex101.com/ does want it to be escaped.

It does not have to be escaped because it is a literal character inside the `[` `]` selector.  I think the only characters you need to escape inside those fancy square brackets are `\` and `]`.  When you try it in regex101.com, make sure you pick the ECMAScript flavor.  I suspect you were looking at it using the PCRE2 flavor (which _does_ require the `/` to be escaped).

Reply:
> I'm wondering why you disallow `/` in the part after the first `.`.

Yeah, I'm wondering that too.  It's probably not necessary.  That said, it also isn't really harmful because FHIR resources and types don't slice (and certainty don't reslice).

But still, I reckon we can get away with one of them `.*` regex black holes here.

Reply:
Fixed in ce58754